data-source/json/test_list_change_evaluates_sections.json

```diff
@@ -3,9 +3,9 @@
     "schema_version": "0.0.1",
     "data_version": "0.0.3",
     "survey_id": "0",
-    "title": "Test ListCollector Evaluate Sections",
+    "title": "Test ListCollector Evaluate Sections for Primary Person",
```

Comment:
I think the "for Primary Person" should be dropped as it's not just for primary person.